associations.rb

```diff
@@ -47,25 +61,46 @@ def generate_migrations
 
 class Hotel < ActiveRecord::Base
   #insert our associations here
- 
+
+  has_many :rooms
+  has_many :bookings, through: :rooms
+
   def to_s
     "#{name} with #{rooms.count} rooms"
   end
+
+  def booked_guests
```

Comment:
Check out [ActiveRecord's `has_many :through` association](http://guides.rubyonrails.org/association_basics.html#the-has-many-through-association). I think it can take care of this.